LogicAppsCustomCodeFramework/README.md

```diff
@@ -0,0 +1,130 @@
+# Logic Apps Custom Code Feature
+
+Custom Code is a new Logic Apps feature in which users can author their own custom code through the usage of .NET Framework code from an Azure Logic Apps workflow.
+This project contains a sample project that utilizes this feature. The contents consist of a Logic App project (WorkflowLogicApp) and an Azure Function App (Function).  
+## Installation
+
+To run this sample, clone the repository to your local machine.
+
+```bash
+git clone https://github.com/Azure/logicapps.git
+```
+To pull the nuget package that contains the WorkflowActionTrigger attribute for your own project, include the phrase below in the csproj file of your project. The link to the myget package that stores this nuget package is listed here: [MyGet]( https://www.myget.org/feed/microsoft-workflowactiontrigger/package/nuget/Microsoft.Azure.Functions.Extensions.Workflows.WorkflowActionTrigger)
+
+
+```bash
+<PackageReference Include="Microsoft.Azure.Functions.Extensions.Workflows.WorkflowActionTrigger" Version="1.0.0" />
+```
+Please ensure you also have the Azure Logic Apps Standard and Azure Functions extensions for Visual Studio Code. This can be downloaded through the Extensions section in Visual Studio Code. For more information on how to install extensions for Visual Studio Code please check here: [EXTENSIONS](https://code.visualstudio.com/docs/editor/extension-marketplace).
+
+
+# Workspace Setup
+To setup the Logic Apps Custom Code feature in a new project, start by creating a Workspace. Due to the nature of this project where there are two distinct applications that need to be ran simultaneously, it is important to create a single VS Code workspace in order to utilize both applications. The steps for doing so are as follows:
+
+1) Create a new folder for your workspace. 
+2) Open this new folder in VS Code and open a terminal by clicking Terminal -> New Terminal (CTRL + SHIFT + 5) 
```

Comment:
isnt is possible to include the sln file also in the sample

Reply:
Yeah I can include the sln file in the sample.